doc/versioning.md

```diff
@@ -0,0 +1,32 @@
+# gRPC-dotnet package versioning policies
+
+## Overview
+
+This document covers versioning policies of the packages produced in this repo:
+
+- Grpc.AspNetCore
+- Grpc.AspNetCore.Server
+- Grpc.AspNetCore.Server.ClientFactory
+- Grpc.Net.Client
+- Grpc.Net.ClientFactory
+- Grpc.Net.Common
+- dotnet-grpc
+
+## Design considerations
+
+### Major version number
+
+The initial major version number was set to 2 to match the release of corresponding Grpc.Core packages. Going forward, the major version will be incremented when there's a need to ingest a major Grpc.Core or ASP.NET Core dependency for breaking changes. Grpc.Core follows these [versioning policies](https://github.com/grpc/grpc/blob/master/doc/versioning.md) and major version is only incremented on rare occasions.
```

Comment:
Minor: Consider "Under [Grpc.Core's versioning policy](https://github.com/grpc/grpc/blob/master/doc/versioning.md) the major version is only..." so that the sentence makes sense with or without the link and the link adds additional value.

Reply:
The doc says "Grpc.Core follows the versioning policy .." , but the policy is gRPC-wide and applies to all languages (including grpc-java, grpc-go and grpc-dotnet). It might be simpler to just say that grpc-dotnet complies with the gRPC-wide versioning policy [link]